proto/motor/v1/bucket.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
I think this file would fit better in the bucket types rather than motor types.

Reply:
migrated